connector/connect/client/jvm/src/test/scala/org/apache/spark/sql/UserDefinedFunctionE2ETestSuite.scala

```diff
@@ -380,23 +387,23 @@ class UserDefinedFunctionE2ETestSuite extends QueryTest with RemoteSparkSession
   }
 
   test("UDAF custom Aggregator - toColumn") {
-    val session: SparkSession = spark
-    import session.implicits._
+    val encoder = Encoders.product[UdafTestInput]
     val aggCol = new CompleteUdafTestInputAggregator().toColumn
-    val ds = spark.range(10).withColumn("extra", col("id") * 2).as[UdafTestInput]
-
+    val ds = spark.range(10).withColumn("extra", col("id") * 2).as(encoder)
     assert(ds.select(aggCol).head() == 135) // 45 + 90
-    assert(ds.agg(aggCol).head().getLong(0) == 135) // 45 + 90
```

Comment:
This does not work in classic either :)... So removing it. I have added a test case below to high light what is supposed to work. cc @xupefei